src/Point/Point.cs

```diff
@@ -48,6 +52,7 @@ public Point(ILogger<Point> logger, IConfiguration configuration, PointMachineSt
 
             _pointState = pointState;
             _pointState.PointPosition = ReportedPointPosition.PointIsInARightHandPositionDefinedEndPosition;
+            _pointState.LatestFinalPointPosition = _pointState.PointPosition;

```

Comment:
shouldn't we initialize _pointState.DegradedPointPosition here to NotDegraded/NotApplicable depending on allPointMachinesCrucial?